### ISSUE TITLE
1000s separator while keeping decimal as is

### ISSUE THREAD

**Issue by mpetkov** [foretagsplatsen/numbro#167]
What would be the syntax to have a format only add a 1000s separator (,) without affecting the decimals?  So for example:

85827 = 85,827
85827.56 = 85,827.56 
85827.56656655 = 85,827.56656655

Essentially I don't care about the decimal places and I just want to add a 1000s separator.

**Comment by BenjaminVanRyseghem** [comment]
> 85827 = 85,827
> 85827.56 = 85,827.56 

See [0,[.]00](http://numbrojs.com/try-me.html?lang=en-US&value=85827.56&format=0%2C%5B.%5D00)

> 85827.56656655 = 85,827.56656655

There is no support yet to keep all the decimals

**Comment by mpetkov** [comment]
Is there a plan to support all decimals in the future?

**Comment by BenjaminVanRyseghem** [comment]
yes, I even thought there was an issue already, but I can't find it :smile:

**Comment by mpetkov** [comment]
Haha ok good to know!  Thanks for answering so quickly.

For the time being I will use this since I won't have more than 8 decimal places:  0,[.][00000000]
